test/src/test/java/org/jenkins/ui/symbol/SymbolJenkinsTest.java

```diff
@@ -15,6 +15,21 @@ public class SymbolJenkinsTest {
     public RealJenkinsRule rjr = new RealJenkinsRule()
             .addPlugins("plugins/design-library.jpi", "plugins/prism-api.jpi", "plugins/bootstrap5-api.jpi");
 
+    @Test
+    @DisplayName("When resolving a symbol where the tooltip contains '$' no error is thrown")
+    public void dollarInToolTipSucceeds() throws Throwable {
```

Comment:
```suggestion
    @Issue("JENKINS-73243")
    public void dollarInToolTipSucceeds() throws Throwable {
```
with `import org.jvnet.hudson.test.Issue;`